dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerMainIntegrationTest.java

```diff
@@ -90,10 +90,11 @@ public void testArtifacts()
       throws IOException, RepositoryException, TransformerException, XMLStreamException {
     try {
       LinkageCheckerMain.main(
-          new String[] {"-a", "com.google.cloud:google-cloud-firestore:0.65.0-beta"});
+          // protobuf-java:3.13.0 in
+          new String[] {"-a", "com.google.cloud:google-cloud-firestore:1.35.2"});
```

Comment:
Why not the latest 2.1.0? It's because the latest version has dependency to protobuf-java:3.13.0, which also have different results between Java 8 and Java 11.